source/designers-guide/edit-newsletter-and-document-templates/index.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: default
+title: Edit newsletter and document templates
+github_link: designers-guide/edit-newsletter-and-document-templates/index.md
+indexed: true
+---
+
+##Table of contents
+-   [Introduction](#introduction)
+-   [Edit newsletter templates](#edit-newsletter-templates)
+    -   [Newsletter directory structure](#newsletter-directory-structure)
+    -   [Change the newsletter logo](#change-the-newsletter-logo)
+    -   [Change the newsletter headline](#change-the-newsletter-headline)
+    -   [Change the newsletters general appearance](#change-the-newsletters-general-appearance)
+    -   [Newsletter snippets](#newsletter-snippets)
+-   [Edit document templates](#edit-document-templates)
+    -   [Documents directory structure](#documents-directory-structure)
+    -   [Adding an article description](#adding-an-article-description)
+
+##Introduction
+
+In this guide you will learn how to modify the default appearance of Shopware's newsletter and document templates. 
+
+<div class="alert alert-warning">
+<strong>Note:</strong> Before you start modifying your newsletter or document templates please keep in mind that you shouldn't apply any changes to the default <code>Bare</code> theme folder. This can cause issues during the next update process and your changes could be overwritten. You should always include your own newsletter and document modifications within your own Shopware theme. If your don't have a custom Shopware theme you should create one for newsletter and document editing.
+</div>
+
+In the following examples we will use a new created theme called `NewsDocExample`:
+
+ ![Newsletter theme](img-example-theme.jpg)
+ 
+ If you want further information about creating custom themes you can read our [Getting started guide](../getting-started/).
+
+##Edit newsletter templates
+
+In the following steps you will learn how to edit Shopware's default newsletter template. If you want an introduction in  general usage of the newsletter module please have a look at our [newsletter wiki tutorial](http://wiki.shopware.com/Newsletter_detail_933.html).
```

Comment:
Please use a link to the english wiki article: http://en.wiki.shopware.com/Newsletter_detail_1159.html